stable_baselines3/common/vec_env/stacked_observations.py

```diff
@@ -75,192 +92,132 @@ def compute_stacking(
         # This includes the vec-env dimension (first)
         stack_dimension = 1 if channels_first else -1
         repeat_axis = 0 if channels_first else -1
-        low = np.repeat(observation_space.low, n_stack, axis=repeat_axis)
-        stackedobs = np.zeros((num_envs,) + low.shape, low.dtype)
-        return channels_first, stack_dimension, stackedobs, repeat_axis
+        stacked_shape = list(observation_space.shape)
+        stacked_shape[repeat_axis] *= n_stack
+        return channels_first, stack_dimension, tuple(stacked_shape), repeat_axis
 
-    def stack_observation_space(self, observation_space: spaces.Box) -> spaces.Box:
+    def stack_observation_space(self, observation_space: Union[spaces.Box, spaces.Dict]) -> Union[spaces.Box, spaces.Dict]:
```

Comment:
I thought about removing it but after a quick check, it is a bit used: https://github.com/search?o=desc&q=stable_baselines3+stack_observation_space&s=indexed&type=Code

so, let's remove it only in 2.x

Reply:
On the contrary, it seems that most of the matches on this search match `def stack_observation_space` that come from the copies of sb3. I can't find any code where this function is actually used.